src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -406,8 +405,12 @@ public virtual AuthenticationProperties ConfigureExternalAuthenticationPropertie
         /// <returns></returns>
         protected async virtual Task<bool> LogResultAsync(bool result, TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
         {
-            Logger.WriteInformation(Resources.FormatLoggingSigninResult(Resources.FormatLoggingResultMessage(methodName,
-                await UserManager.GetUserIdAsync(user)), result));
+            // Check if log level is enabled before creating the message.
```

Comment:
I aligned the experience with the logging guidelines given at https://github.com/aspnet/Logging/wiki/Guidelines which says `Warning` is for unexpected flow in the application. In SignInManager, the result is bool for operations that are mostly read-only and most of the time it is an expected value. You might argue that locking out user might be an unexpected error but it is a known result. Hence I choose to log all flows of SignInManager. In contrast, adding an existing role to the database or creating a user with existing username is an error in the happy flow hence I chose it as Warning

I am open to this being changed since this is subjective. From a uniformity perspective yes, we can the same in SignInManager as well.